maskit/examples/load_iris.py

```diff
@@ -0,0 +1,36 @@
+from pennylane import numpy as np
+from sklearn import datasets
+from maskit.examples.utils import one_hot
+
+np.random.seed(42)
+
+MAX_TRAIN_SAMPLES = 150
+
+
+def load_iris(wires, params):
+    train_size = MAX_TRAIN_SAMPLES
+    if "train_size" in params:
+        train_size = min(params["train_size"], MAX_TRAIN_SAMPLES)
```

Comment:
You can merge these into one. Not sure if it helps readability, though.
```suggestion
    train_size = min(params.get("train_size", MAX_TRAIN_SAMPLES), MAX_TRAIN_SAMPLES)
```

Reply:
Yes, I had the same idea, but wasn't sure about the readability. Changed it to the one-liner now.